web/src/Protected.jsx

```diff
@@ -27,6 +27,8 @@ import { AppProviders } from "./context/app";
 export default function Protected() {
   const { isLoggedIn } = useAuth();
 
+  if (isLoggedIn === undefined) return;
```

Comment:
I think this deserve a bit of comment in code why we do it as it is not obvious especialyl with combination of condition below.

Reply:
Well, it is a rather typical pattern in Reac/JavaScript, but a comment won't hurt.